lib/promscrape/scrapework.go

```diff
@@ -500,6 +504,7 @@ func (sw *scrapeWork) processDataOneShot(scrapeTimestamp, realTimestamp int64, b
 	am := &autoMetrics{
 		up:                        up,
 		scrapeDurationSeconds:     scrapeDurationSeconds,
+		scrapeResponseSize:        float64(len(bodyString)) + float64(unsafe.Sizeof(bodyString)),
```

Comment:
Could you elaborate why we are including size of `bodyString` here as well?
As far as I can see, the size of response in bytes will be equal to `len(bodyString)` but maybe I'm missing something.

Reply:
updated this